api/pkg/cli/cmd/info/info.go

```diff
@@ -194,11 +195,19 @@ func (opts *options) run() error {
 	out := opts.cli.Stream().Out
 
 	resVersion := resource.(hub.ResourceWithVersionData)
+
+	var deprecated bool
+	if resVersion.Deprecated != nil {
```

Comment:
cant we use bool now

Reply:
+1

Reply:
we can't use ,In response for non deprecated version of a resource doesn't contains deprecated filed so needs to check it